### DISCUSSION TITLE
After Windows automatically updates, crazy pop-ups pop up on the home screen, and then the program crashes

### DISCUSSION THREAD

**Discussion by limit4ever** [valinet/ExplorerPatcher#2530] [Q&A]
After Windows automatically updates, crazy pop-ups pop up on the home screen, and then the program crashes

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
